### ISSUE TITLE
Accessibility - missing form labels on "Add Dataverse" page

### ISSUE THREAD

**Issue by kaitlinnewson** [IQSS/dataverse#7566]
Hi IQSS team,

This issue was tested on Harvard Dataverse (5.3) and on Scholars Portal Dataverse (5.1.1).

On the "Add Dataverse" page, there are two form fields that don't have the labels properly associated with them: "Host Dataverse" and "Email".

For "Host Dataverse", the label is set to `for=dataverseForm:selectHostDataverse`, but the input id is `dataverseForm:selectHostDataverse_input` (the proper id seems to be on the span element).

For "Email", the label is set to `for=dataverseForm:contactEmail`, but the input id is `dataverseForm:j_idt265:0:contactEmail`.

**Comment by djbrooke** [comment]
Thanks @kaitlinnewson, we'll take a look

**Comment by mheppler** [comment]
Was able to identify the missing input labels looking at demo (5.3) using the SiteImprove extension tool for Chrome.

<img width="951" alt="Screen Shot 2021-02-10 at 1 45 47 PM" src="https://user-images.githubusercontent.com/687227/107557333-76191d80-6ba7-11eb-8f85-ac4142c4459c.png">

In addition to the Host Dataverse and Email form inputs, there were "no description" errors reported for the Browse/Search Facets picklist, as well as the Contact Support popup form linked from the navbar. I was able to fix the contact popup, but there appears to be an issue with the PrimeFaces picklist component, as it does not properly reference the labels associated with "hidden-accessible" form input dynamically created. This will need to be revisited when we next update PrimeFaces.
